src/arch/arm/tlb.cc

```diff
@@ -58,8 +58,38 @@ namespace gem5
 
 using namespace ArmISA;
 
+TlbEntry*
+TLB::Table::accessEntry(const KeyType &key)
+{
+    auto entry = findEntry(key);
```

Comment:
This could be:
`
if (!key.functional) {
    entry = accessEntry(key);
} else {
    entry = findEntry(key);
}
`

Reply:
Unless I am missing something the first case would imply an endless recursion

Reply:
Ah, right. Missed that we are in the same function. 😂